bindings/matrix-sdk-ffi/src/room/mod.rs

```diff
@@ -615,6 +617,54 @@ impl Room {
         Ok(())
     }
 
+    // Tchap-specific : invite_users_by_email
+    pub async fn invite_user_by_email(&self, email_to_invite: String) -> Result<(), ClientError> {
+        self.invite_users_by_email(vec![email_to_invite]).await?;
+        Ok(())
+    }
+
+    pub async fn invite_users_by_email(
+        &self,
+        emails_to_invite: Vec<String>,
+    ) -> Result<(), ClientError> {
+        // Get openIdToken
+        let user_id = self.inner.own_user_id().to_owned();
+        let open_id_response = self.inner.client().send(OpenIdRequest::new(user_id)).await?;
+
+        // Send identity_account_register
+        let account_register_response = self
+            .inner
+            .client()
+            .send(matrix_sdk_tchap::request::identity_account_register::v1::Request::new(
+                open_id_response,
+            ))
+            .await?;
```

Comment:
What's the behavior if the user is not authorized, or no token is given back ? Maybe we should break early in those cases